### PR TITLE
Updated error msg and post and patch req body

### PR DESCRIPTION
**Pull request by asching7108**
- Updated POST and PATCH pets request body to remove the extra layer
- Changed error response to avoid this error in the frontend: 'SyntaxError: Unexpected token N in JSON at position 0'

### CODE REVIEW

**Review by xHoudek — COMMENTED** [review]
Looks good! Will this break existing functionality in the frontend? Or is it safe to merge now?

**Comment by asching7108** [comment]
> Looks good! Will this break existing functionality in the frontend? Or is it safe to merge now?

Related frontend changes are included in [Pm 25 pet page](https://github.com/asching7108/purrfect-match/pull/8), so it might be temporarily broken but should work after the frontend is merged

**Review by xHoudek — APPROVED** [review]
Don't merge until PM-25 is approved/merged

**Pull request merged**
